assets/scripts/build_package_gallery.py

```diff
@@ -90,7 +92,8 @@ def write_results_to_disk(path: str, package_manifests: List[dict]) -> None:
         package_manifest: A list of dictionaries that represents contributor package manifests
     """
     with open(path, "w") as outfile:
-        json.dump(package_manifests, outfile, indent=4)
+        package_manifests_serialized = convert_to_json_serializable(package_manifests)
```

Comment:
We need to make sure this is json serializable using our custom method that handles complex types before serializing using the built in json.dump.